### ISSUE TITLE
tool panel list

### ISSUE THREAD

**Issue by davidchristiany** [ifb-git/ProteoRE#253]
It is possible to order tools in the tool panel section thanks to integrated_tool_panel.xml file in the galaxy root directory.
But you can't comment or remove tools or different versions of tools in this file because it is generated at each start of the instance.
So every version of (active) tools are available if they are installed.

Different versions of tools are "concatenated" if they are on following lines in the integrated_tool_panel.xml file.

**Comment by combesf** [comment]
I install a new tool on the proteore.org instnace via graphical "admin" page. 
I than modified by hand the "integrated_tool_panel.xml" to reorder the tool at the right subsection, 
but it seems not be taken into account.
Do I have to restart the instance ? @vloux @davidchristiany 
Thanks.

**Comment by davidchristiany** [comment]
Yes you have to restart your instance to update your tool panel with "integrated_tool_panel.xml"

**Comment by combesf** [comment]
I just restarted. The "integrated_tool_panel.xml" is as it should,but online the tool I installed is still at the top of the tool panel list and not in the section "GO terms analysis" ..... ?

**Comment by davidchristiany** [comment]
It's odd, usually it works. I checked the integrated_tool_panel.xml file and it looks fine. 
Did you change the file again after restarting ? Because the file and the tool panel don't match.
This file is generated at each start of the instance so I am not sure of what happened here. 
I would restart again to be sure.
What command did you use to restart ? whith the proteore user I presume.

**Comment by vloux** [comment]
TODO : keep a trace of integrated_tool_panel.xml in the repo

**Comment by combesf** [comment]
> It's odd, usually it works. I checked the integrated_tool_panel.xml file and it looks fine.
> Did you change the file again after restarting ? Because the file and the tool panel don't match.
> This file is generated at each start of the instance so I am not sure of what happened here.
> I would restart again to be sure.
> What command did you use to restart ? whith the proteore user I presume.

The problem was not in relation with the integrated_tool_panel.xml file. 
Proteore instance didn't restrart due to a very different pb.

**Comment by combesf** [comment]
I have a question, not about the order of the section and the tools in the tool panel, 
but about the little comments after the name of the tool. 
For example: for the tool 'volcano plot' : it is not on the GitHuv as it is a "public" tool. 
Nevertheless I Have to change from : 
**Volcano plot** create a volcano plot
to : 
Draw a volcano plot 

I don't see where to read "Draw a volcano plot" for this to be written in the tool panel. 

I checked on proteore the 
integrated_tool_panel.xml
config/tool_conf.xml*
config/seh_tool_conf.xml*

but I saw nowhere where this is set .... 

Thanks !

**Comment by vloux** [comment]
The names of the tools are defined by the authors. For those of which we are not the author, the only way to change them is to fork them (or ask the author) I think …

**Comment by combesf** [comment]
Got it ; thank you @vloux

**Comment by vloux** [comment]
Documentation : https://docs.galaxyproject.org/en/master/admin/tool_panel.html
